flaskr/app.py

```diff
@@ -7,7 +7,9 @@ def register_models(app):
 
 def register_blueprints(app):
     from .bojangles import app as bojangles_bp
+    from .api import app as api_bp
```

Comment:
Please import these at the top scope the file - import errors should be uncovered eagerly, not lazily/dynamically.